tenets/codelingo/code-review-comments/context-first-arg/expected.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I'm getting different output for expected.json. Now I'm questioning my own methodology. Must ask blake / jess / reuben which directory this should be run from to ensure unit tests pass

```
diff --git a/expected.json b/expected.json
index 9719649..3171cba 100755
--- a/expected.json
+++ b/expected.json
@@ -1,14 +1,14 @@
 [
   {
    "Comment": "Most functions that use a Context should accept it as their first parameter.",
-   "Filename": "effective-go/context-first-arg/example.go",
+   "Filename": "example.go",
    "Line": 9,
-   "Snippet": "func F(ctx context.Context, a int)               {}\nfunc F(b int, ctx context.Context, a int)        {}\nfunc F(c int, b int, ctx context.Context, a int) {}\n"
+   "Snippet": "\nfunc F(ctx context.Context, a int)               {}\nfunc F(b int, ctx context.Context, a int)        {}\nfunc F(c int, b int, ctx context.Context, a int) {}"
   },
   {
    "Comment": "Most functions that use a Context should accept it as their first parameter.",
-   "Filename": "effective-go/context-first-arg/example.go",
-   "Line": 8,
-   "Snippet": "\nfunc F(ctx context.Context, a int)               {}\nfunc F(b int, ctx context.Context, a int)        {}\nfunc F(c int, b int, ctx context.Context, a int) {}"
+   "Filename": "example.go",
+   "Line": 10,
+   "Snippet": "func F(ctx context.Context, a int)               {}\nfunc F(b int, ctx context.Context, a int)        {}\nfunc F(c int, b int, ctx context.Context, a int) {}\n"
   }
  ]
\ No newline at end of file
```